src/protocol/graphics.ts

```diff
@@ -19,6 +19,8 @@ import { setCanvas, setEventsForType, setVideoUrl } from "ui/actions/app";
 import { setPlaybackPrecachedTime, setPlaybackStalled } from "ui/actions/timeline";
 import { getPlaybackPrecachedTime, getRecordingDuration } from "ui/reducers/timeline";
 
+const FIVE_SECONDS = 5000;
```

Comment:
lets use a semantic variable like `MIN_VIDEO_DURATION`

at some point we'll change the value and this will be a lie